src/agent/nodes/schedule.py

```
@@ -111,7 +111,7 @@ class ScheduleResponse(BaseModel):
 
 
 def schedule_agent(state: AgentState) -> AgentStateUpdate:
-    local = state.config_state.model_copy(deep=True)
+    local = clone_for_phase(state)
```

Ruff (0.16.1)

[error] Undefined name `clone_for_phase`

(F821)